package.json

```diff
@@ -26,7 +26,8 @@
     "build:js:edit": "node ./notebook/build.js edit/js/main.js edit/js/main.bundle.js",
     "build:js:tree": "node ./notebook/build.js tree/js/main.js tree/js/main.bundle.js",
     "build:js:auth": "node ./notebook/build.js auth/js/main.js auth/js/main.bundle.js",
-    "build:js:terminal": "node ./notebook/build.js terminal/js/main.js terminal/js/main.bundle.js"
+    "build:js:terminal": "node ./notebook/build.js terminal/js/main.js terminal/js/main.bundle.js",
+    "prepublish": "npm run bower && npm run build"
```

Comment:
Yeah, I found that weird too but it only does that for dev installs. That ends up being a positive benefit as it ensures that on a dev copy you're looking at the built from source version rather than some old artifact.